services/core-web/src/components/Forms/permits/GeneratePermitForm.js

```diff
@@ -48,6 +50,30 @@ export const GeneratePermitForm = (props) => (
             />
           </Col>
         </Row>
+        <Row gutter={32}>
+          <Col xs={24} md={12}>
+            <Field
+              id="now_number"
+              name="now_number"
+              label="Application Number"
+              required
```

Comment:
does 'required' do anything ?  ` validate={[required]}` should handle that. 
Same with the Field below.

Reply:
'required' is used on every field on this page, I'll take look if it does anything